docs/architecture/components/1_batcher.md

```diff
@@ -0,0 +1,13 @@
+# Batcher
+
+Batcher is the Entity in charge of recieving User's proofs.
+
+The Batcher will recieve proofs from different Users, build a merkle root from these proofs, upload the proofs to an S3 (so that the operators can download them), and submit this information to the [Aligned Service Manager](./3_service_manager_contract.md).
+
+To avoid trust assumptions, the Batcher has some specific interesting mechanisms;
+
+So that the User is 100% sure that his proof was included in a batch, the Batcher will send to each User the Merkle Proof (or Merkle Path). With this, the User can rebuild the Merkle Root starting from his proof, thus verifying his proof was actually included in the batch.
```

Comment:
```suggestion
So that the User is sure that his proof was included in a batch, the Batcher will send to each User the Merkle Proof (or Merkle Path). With this, the User can rebuild the Merkle Root starting from his proof, thus verifying his proof was actually included in the batch.
```